db/schema.rb

```diff
@@ -116,6 +116,11 @@
     t.datetime "updated_at"
   end
 
+  create_table "concept_child_relations", force: true do |t|
+    t.integer "parent_id"
+    t.integer "child_id"
+  end
+
```

Comment:
@marcellosachs This `concept_child_relations` keeps coming back because this table only exists in your local db. there is no model or migration for it, would you mind dropping that table from your local dev box and taking this out?